### PR TITLE
Add livestream link.

### PR DESCRIPTION
**Pull request by mdparker**

### CODE REVIEW

File: 2017/index.dd [hunk 1]
```diff
@@ -4,6 +4,8 @@ $(T h1,,DConf 2017$(BR) Bigger, Badder, and Berliner!)
 
 $(T h2,,Please join us at DConf 2017, the international conference of the D programming language in Berlin, Germany, May 4-7 2017.)
 
+$(P DConf 2017 is now underway! If you aren't with us in Berlin, be sure to join us via the livestream at $(HTTP youtube.com/watch?v=MqrJZg6PgnM&utm_content=buffercc4c1&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer, YouTube)).
```
Comment (CyberShadow) [inline 2017/index.dd]: I don't think there needs to be anything in the URL after the first ampersand (though it doesn't break anything)

**Comment by andralex** [comment]
@CyberShadow will this be auto-deployed?

**Comment by CyberShadow** [comment]
Auto-deploy is done through Travis, which is currently clogged by some DMD pulls.

**Comment by CyberShadow** [comment]
We aborted the DMD Travis tasks (https://github.com/dlang/dmd/pull/6736 which was merged anyway), so the deploy should be running now

**Comment by CyberShadow** [comment]
Deployed :)

File: 2017/index.dd (hunk 1 above)
Comment (andralex) [inline 2017/index.dd]: The closing colon is also misplaced, will edit the file.

**Comment by mdparker** [comment]
Sorry, guys. I put the period outside the P tag.

**Pull request merged**